exec.go

```diff
@@ -46,14 +55,43 @@ func (exec *EXEC) Exports() modules.Exports {
 }
 
 // Command is a wrapper for Go exec.Command
-func (*EXEC) Command(name string, args []string, option CommandOptions) string {
+func (*EXEC) Command2(name string, args []string, option CommandOptions) CommandReturn {
```

Comment:
> But back to your comment, It's not clear to me if it would be best to avoid the extra method and keep completely backwards compatible, or should I still keep this new method idea, probably with a better name, (one that could become the default in the future?)

The suggestions of the callback is not to avoid the extra method. There are other reasons:
- It is simple to implement: one `if` stament plus some boiler plate for invoking the callback
- it avoids multiple conditionals if we want to further tweak the logic (for example, suppressing the lot altogether)
- it opens other possibilities in the future
- it seems more idiomatic to me, as this callbacks on events are quite common in JS (but I acknowledge this is debatable) 

Adding a second method just seem simpler on the context of the requirement you want to cover now, but not in general.

But as I said before, I'm not strongly against this option. golang `exec` package has an example of such convenience functions that cover specific use cases: `CombinedOutput` executes a command and returns the combined `stdout` and `stderr`. In our case, `Command` would be the convenience function that cover the simplest use case.

And I agree a more descriptive name is required for the new function.

And if we follow this path, then I think the flag you suggested for not failing on error is a good addition.